apps/api/pages/api/bookings/_get.ts

```diff
@@ -203,6 +207,25 @@ async function handler(req: NextApiRequest) {
     }
     args.where = buildWhereClause(userId, attendeeEmails, [], []);
   }
+
+  if (dateFrom && dateTo) {
+    args.where = {
+      ...args.where,
+      startTime: { gte: dateFrom },
+      endTime: { lte: dateTo },
+    };
+  } else if (dateFrom) {
+    args.where = {
+      ...args.where,
+      startTime: { gte: dateFrom },
+    };
+  } else if (dateTo) {
+    args.where = {
+      ...args.where,
+      endTime: { lte: dateTo },
+    };
+  }
```

Comment:
Could probably do something like this to keep it clean, more readable and remove the need for extra `else-ifs` , thoughts @emrysal 🤔?
```suggestion
  if (dateFrom) {
    args.where.startTime = { gte: dateFrom };
  }
  if (dateTo) {
    args.where.endTime = { lte: dateTo };
  }
```

Reply:
🔥